apps/admin-gui-e2e/src/integration/group-management-perun_admin.spec.js

```diff
@@ -206,16 +226,18 @@ context('Actions', () => {
 
     cy.intercept('**/authzResolver/unsetRole/**').as('unsetRole')
       .get('[data-cy=advanced-settings]')
-        .click()
+      .click()
       .get('[data-cy=managers]')
-        .click()
+      .click()
       .get(`[data-cy=${dbGroupManager}-checkbox]`)
-        .click()
+      .click()
       .get('[data-cy=remove-manager-button]')
-        .click()
+      .click()
       .get('[data-cy=remove-manager-button-dialog]')
-        .click()
+      .click()
       .wait('@unsetRole')
+      .intercept('**/groupsManager/getRichGroupByIdWithAttributesByNames**').as('getRichGroup')
+      .wait('@getRichGroup')
```

Comment:
```suggestion
      .intercept('**/groupsManager/getRichAdmins**').as('getRichAdmins')
      .get('[data-cy=remove-manager-button-dialog]')
      .click()
      .wait('@unsetRole')
      .wait('@getRichAdmins')
```
I dunno why would you should wait for getRichGroup - you should wait for get rich admins